content/articles/account-collaborator.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: Differences between Account Collaborator and Account Membership
+excerpt: The differences between an Account Collaborator and an Account Member.
+categories:
+- Account
+---
+
+# Account Collaborators and Account Memberships
```

Comment:
We rarely refers to "Account Membership". Not even in https://support.dnsimple.com/articles/account-users/.

Should we perhaps use:

> Account Collaborators and Multi-User Accounts

Reply:
That makes perfect sense to me, I'll change 👍

Reply:
This is now fixed.